### ISSUE TITLE
Warning Output from GitHub Actions

### ISSUE THREAD

**Issue by jacobwoffenden** [docker/build-push-action#703]
### Troubleshooting

Before submitting a bug report please read the [Troubleshooting doc](https://github.com/docker/build-push-action/blob/master/TROUBLESHOOTING.md).

### Behaviour

```
Warning: The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

As per https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

> Action authors who are using the toolkit should update the @actions/core package to v1.10.0 or greater to get the updated saveState and setOutput functions.

I can see the latest v3 release (v3.1.1) uses `@actions/core@^1.9.0` https://github.com/docker/build-push-action/blob/v3.1.1/package.json#L31, and `master` has already been updated to `@actions/core@^1.10.0` via https://github.com/docker/build-push-action/pull/695

#### Steps to reproduce this issue

1. use `docker/build-push-action@v3` to build an image

#### Expected behaviour

> Tell us what should happen

No warnings

#### Actual behaviour

> Tell us what happens instead

Warning output from GitHub Actions

### Configuration

* Repository URL (if public): 
* Build URL (if public): 

```yml
      - name: Build Container
        id: build
        uses: docker/build-push-action@v3
        with:
          file: Containerfile
          context: .
          platforms: linux/${{ matrix.platform }}
          load: true
          tags: |
            container:latest
```

### Logs

> Download the [log file of your build](https://docs.github.com/en/actions/managing-workflow-runs/using-workflow-run-logs#downloading-logs) and [attach it](https://docs.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests) to this issue.

**Comment by crazy-max** [comment]
Should be good with latest release

**Comment by jacobwoffenden** [comment]
Are you using any other Docker actions @kedare ? I still see this warning output from a workflow that uses the following actions:

- `docker/setup-qemu-action` 
- `docker/setup-buildx-action`
- `docker/build-push-action`

Frustratingly the actions UI doesn't specify which one causes it

**Comment by kedare** [comment]
I just use the build push one

**Comment by crazy-max** [comment]
@kedare This has been fixed for `docker/build-push-action@v3.2.0` and `docker/build-push-action@v3` and all other Docker actions as well (bake-action, setup-buildx-action, setup-qemu-action, metadata-action).

**Comment by crazy-max** [comment]
> Frustratingly the actions UI doesn't specify which one causes it

You can find out which step generates a warning by clicking on it.
